x/nodes/keeper/keeper.go

```diff
@@ -107,8 +107,6 @@ func (k Keeper) ConvertState(ctx sdk.Ctx) {
 		panic(err)
 	}
 	k.Cdc.SetUpgradeOverride(true)
-	// custom logic for minSignedPerWindow
```

Comment:
I'm not familiar with this whole manual update for MinSignedPerWindow so it's the only piece that's giving me a bit of "unease" but I don't know how to confirm/validate it.

Is your confidence level high on these changes?

Reply:
> The ONLY concern I have is related to the cleanup you did related to the MinSignedPerWindow. I don't know why we had this before and wanted to understand your confidence level on it?

For the details, please check the commit message [here](https://github.com/pokt-network/pocket-core/pull/1580/commits/9a11177eb35da369fd0e48566dd7a888303e8558).

The reason why we have this "custom logic" is the param `SignedBlocksWindow` is a proportion (= 0.6 in the mainnet), while the function `SignedBlocksWindow` is expected to return the number of blocks in a window (= 6 in the mainnet).  This custom logic converts the proportion to the number of blocks by multiplying `SignedBlocksWindow`.  This patch reconciles this discrepancy under the name `SignedBlocksWindow`.

The behavior is clear and the regression risk is low.